versions/3.0.md

```diff
@@ -53,12 +53,12 @@ Additional utilities can also take advantage of the resulting files, such as tes
 		- [Callback Object](#callbackObject)
 		- [Headers Object](#headersObject)
 		- [Example Object](#exampleObject)
```

Comment:
That seems like a reasonable approach.  I must admit, I did like the "Described Example" moniker but `Example Value` is more accurate than `Example Object` for the current usage.